e2e/test/scenarios/dashboard-filters/dashboard-filters-text-category.cy.spec.js

```diff
@@ -102,7 +102,7 @@ describe("scenarios > dashboard > filters > text/category", () => {
         waitDashboardCardQuery();
         filterWidget()
           .eq(index)
-          .contains(single ? value : /\d selections/);
+          .contains(single ? value.replace(/"/g, "") : /\d selections/);
```

Comment:
This is to handle the quotes that were added [here](https://github.com/metabase/metabase/pull/43293/files#diff-9059336d6e3656eddeb97cf6e0b3ca73bb9c1d2c6d8fa10dbb1141005f09a916R24).